src/index.html

```diff
@@ -8,6 +8,18 @@
 	<meta name="description" content="The home of dojo. Documentation. Examples. Community.">
 	<link rel="icon" href="./assets/favicon.ico" />
 	<link rel="stylesheet" type="text/css" href="./assets/prism.css" />
+
+	<!-- Google Analytics -->
+	<script type="text/javascript">
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+		ga('create', 'UA-3242977-15', 'auto');
```

Comment:
Just confirming that this code should be different than the current site:

```
ga('create', 'UA-3242977-2', 'auto');
```

Reply:
Its the same as the deployed version of dojo.io
![image](https://user-images.githubusercontent.com/1388138/56907775-0047ce80-6a73-11e9-9e6c-fe166e242591.png)

Reply:
Ah, don't mind me, I was looking at https://sitepen.com.